### PR TITLE
docs improvements and hot fixes, to match new document parsing strategy   - bulk QA script updated perf included

### PR DESCRIPTION
**Pull request by snova-jorgep**

### CODE REVIEW

**Comment by snova-jorgep** [comment]
@snova-kwasia

File: utils/parsing/config.yaml [hunk 1]
```diff
@@ -31,7 +31,7 @@ partitioning:
   metadata_include: []
   partition_endpoint: 'http://localhost'
   unstructured_port: 8005
-  partition_by_api: True
```
Comment (snova-kwasia) [inline utils/parsing/config.yaml:34]: @snova-jorgep This does actually need to be true but we need to change 
-  'set to true if using Docker container' to
- 'set' to true if using API server.

We are still parsing via API server, it's just that it's a local one vs docker

**Review by snova-kwasia — CHANGES_REQUESTED** [review]
Hi @snova-jorgep looks good, just one minor change re the config.yaml in parser which I've explained. Let me know if you have any questions. Aside from that looks good to go! Thanks

File: utils/parsing/config.yaml (hunk 1 above)
Comment (snova-jorgep) [reply utils/parsing/config.yaml:34]: fixed

**Comment by snova-jorgep** [comment]
Added minor fixes and improvements

**Comment by snova-kwasia** [comment]
Looks good @snova-jorgep . Merged!

**Pull request merged**